app/controllers/users/courses_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Users::CoursesController < ApplicationController
+  ALLOWED_TARGETS = %w[rails_course front_end_course other_courses].freeze
```

Comment:
細かいですが、course モデルにデータがあるのでそちらから生成するといいと思いました🙇‍♂️
現状だとコース変更があった際にモデルとコントローラの両方をメンテする必要があり、管理が大変になりそうかと思います🙇‍♂️

```suggestion
  ALLOWED_TARGETS = Course.course_names.keys.map(&:to_s).freeze
```

Reply:
と思いましたが、どのコースを許可するかはユーザのコースコントローラで決めても問題ない気がしました🙇‍♂️お騒がせしました🙇‍♂️